kubernetes/resource_kubernetes_ingress_test.go

```diff
@@ -147,6 +148,27 @@ func TestAccKubernetesIngress_InternalKey(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesIngress_WaitForLoadBalancerGoogleCloud(t *testing.T) {
+	var conf api.Ingress
+	name := fmt.Sprintf("tf-acc-test-%s", acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:      func() { testAccPreCheck(t); skipIfNoGoogleCloudSettingsFound(t) },
```

Comment:
This will only run in Google Cloud for the moment. EKS and Azure have a bunch of extra steps to get the Load balancer integration working with Ingresses, whereas it's baked into GKE.

The test will work in `kind` if you uncomment this skip,